### ISSUE TITLE
Custom styles for jQuery UI elements

### ISSUE THREAD

**Issue by jeremyboggs** [rcmapp/CBW#39]
The site references a CSS file for jQuery UI. There are likely a lot of unused styles in this file, and the styles themselves don't quite match the aesthetics of the site. Let's drop this style sheet in favor of our own styles, written for the things we specifically include with jQuery UI. That list, so far, includes:

* Autocompletes on search pages.
* ?

**Comment by rcmapp** [comment]
Yes please.  That makes sense. Could I look at the (unneeded) styles that
we have now?  So that I can understand the kinds of things that such a
style file (?) can/should have?

Jeremy, I can't tell you how much I appreciate the professionalism and
expertise you are bringing to this work.  As you probably know I was out of
the country for three weeks and then came home with a terrible sinus
infection, so I haven't been very responsive to your GitHub work, and I'm
sorry for that.

I like to think that in general when I thank someone, I'm being perfectly
sincere; but in this case I want to add an *extra* dose of sincerity if
that's possible.

All best,
Rennie

Rennie Mapp
Project Manager, Collective Biographies of Women, IATH
Content Director, UVa Knowledge Base, SHANTI
University of Virginia
(717)713-1204
